api/mysql.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package api;
+
+import "google/api/annotations.proto";
+
+message MySQLNode {
+    reserved 1, 2; // id and type
+
+    string region = 3;
+    string name = 4;
```

Comment:
What are the region and name there?

Reply:
we have host and port in `MySQLService`, should we duplicate it here?